subprojects/ce-login/celogin/src/CeLoginJson.cpp

```diff
@@ -270,10 +270,6 @@ CeLoginRc CeLogin::decodeJson(const char* jsonStringParm,
                 ParseAdminResetFields(sJsmnState, sVersion, serialNumberParm,
                                       serialNumberLengthParm, decodedJsonParm);
         }
-        else
```

Comment:
Because github is stupid and I can't comment on line 267 I will comment here that line 267 needs to be removed as well right?  To allow the fields to be parsed without the replayid?

Reply:
Good catch; will update.